src/common/hooks/useGeolocation.ts

```diff
@@ -59,31 +59,30 @@ function fetchGeolocationData() {
 // Use this mock to simulate fetchGeolocationData, other data can be obtained from
 // https://ip-api.com/#<IP_ADDRESS> (https://ip-api.com/#8.8.8.8)
 
-/* tslint:disable */
-function mockGeolocationData() {
-  return Promise.resolve({
-    query: '8.8.8.8',
-    status: 'success',
-    continent: 'North America',
-    continentCode: 'NA',
-    country: 'United States',
-    countryCode: 'US',
-    region: 'VA',
-    regionName: 'Virginia',
-    city: 'Ashburn',
-    district: '',
-    zip: '20149',
-    lat: 39.03,
-    lon: -77.5,
-    timezone: 'America/New_York',
-    offset: -18000,
-    currency: 'USD',
-    isp: 'Google LLC',
-    org: 'Google Public DNS',
-    as: 'AS15169 Google LLC',
-    asname: 'GOOGLE',
-    mobile: false,
-    proxy: false,
-    hosting: true,
-  });
-}
+// function mockGeolocationData() {
```

Comment:
would be nice to add a quck comment about why this is commented out when you get the chance!

Reply:
theres a comment at the top of the page: 

> Note: geolocation does not work locally. To test locally, replace fetchGeolocationData() with mockGeolocationData() in the effect.